src/make-scm-group-role.js

```diff
@@ -0,0 +1,46 @@
+import editRole from './util/edit-role';
+import {getProjects, hgmoPath} from './util/projects';
+
+module.exports.setup = (program) => {
+  return program
+    .command('make-scm-group-role <group>')
+    .option('-n, --noop', 'Don\'t change roles, just show difference')
+    .description('create or update a mozilla-group:scm_foo role, based on production-branches.json');
+};
+
+module.exports.run = async (group, options) => {
+  var taskcluster = require('taskcluster-client');
+  var chalk = require('chalk');
+  var auth = new taskcluster.Auth();
+
+  // find the list of projects with this group
+  var projects = await getProjects();
+  var projectsWithGroup = Object.keys(projects)
+    .filter(p => projects[p].access === group)
+    .map(p => projects[p]);
+
+  var roleId = 'mozilla-group:' + group;
+  var scopes = projectsWithGroup.map(project => {
+    let path = hgmoPath(project);
+    return `assume:repo:hg.mozilla.org/${path}:*`;
+  });
+  // not sure what this is
+  scopes.push('queue:get-artifact:private/docker-worker/*');
```

Comment:
I really do not think this is necessary, I don't recall any private artifacts there that we might actually use somewhere.  I say remove it.  What could possibly go wrong?